packages/docs-next/documentation/index.md

```diff
@@ -38,7 +38,7 @@ yarn add @oruga-ui/oruga-next
 ```
 
 ```html [Cdn]
-<script src="https://unpkg.com/@oruga-ui/oruga-next/dist/oruga.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/@oruga-ui/oruga-next/dist/oruga.min.js"></script>
```

Comment:
The problem is the `.min'.
We now ship the minified version of the main package by default. 
You need to remove the `.min' extension.

"https://unpkg.com/@oruga-ui/oruga-next/dist/oruga.js" is working.